model/minsky.cc

```diff
@@ -265,8 +265,14 @@ namespace minsky
     canvas.setItemFocus(model->addGroup(g));
     m.populateGroup(*g);
     // Default pasting no longer occurs as grouped items or as a group within a group. Fix for tickets 1080/1098    
-    auto p = model;
-	p->moveContentsInSelection(*g);
+    auto p = minsky().model;
+	for (auto i: p->items)	
+	    if (!dynamic_cast<IntOp*>(i.get())) canvas.selection.toggleItemMembership(i);   
+	p->moveContents(*g);
+	auto q = model;
+	for (auto i: q->items)
+	  canvas.selection.toggleItemMembership(i); 	
```

Comment:
Assuming selection is empty, this adds all model items to the selection. Again not what we want.

Reply:
Mmmmm, I'm seeing really bizarre behaviour. When I use only "auto p=canvas.model" or "auto p=minsky().model", a subset of items remain selected after "p->moveContents(*g)" in most of the examples I've tested. But they are not the items I have selected for copying/cutting and pasting. The only time I get the items I've selected (and very weirdly also the two first variables on the bottom of a Godley icon???) after pasting, is when I first loop over items in minsky().model and afterwards canvas.model. I agree that is bizarre, but I don't understand why it happens.